### ISSUE TITLE
Self-Voicing Options for Layered Approach 

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#202]
@jessegreenberg, I have fleshed out the input details, level options, and provided a walkthrough for the Basic Level (Level 0) in the design doc. The design plan starts here at [Self-Voicing Options for Layered Approach](https://docs.google.com/document/d/1XFTD2HTcGCEOP2yQ9_2SkEeXmCceZYUCmNeyiJfcCHs/edit#heading=h.or6pn8kfgkr) 

I think the important implementation details are in these tables:
- [Table: Self-Voicing Options Menu (sketches)](https://docs.google.com/document/d/1XFTD2HTcGCEOP2yQ9_2SkEeXmCceZYUCmNeyiJfcCHs/edit#heading=h.pbh4fmwxyh1r) - see option 2
- [Table: Input Methods](https://docs.google.com/document/d/1XFTD2HTcGCEOP2yQ9_2SkEeXmCceZYUCmNeyiJfcCHs/edit#heading=h.gwmun61y4his)
- [Table: Self-Voicing Option Levels](https://docs.google.com/document/d/1XFTD2HTcGCEOP2yQ9_2SkEeXmCceZYUCmNeyiJfcCHs/edit#heading=h.1dj82k56drw2)
- [Table: Level 0 (Basic) Actions, Visuals & Self-Voicing content](https://docs.google.com/document/d/1XFTD2HTcGCEOP2yQ9_2SkEeXmCceZYUCmNeyiJfcCHs/edit#heading=h.lgp2nc4eezwj)

@jessegreenberg, have a look and let's discuss any thing that is not clear in the design document. I'll start working on a Level 1 walk through that should be more aligned with the PDOM strings.

**Comment by terracoda** [comment]
@jessegreenberg, the implementation of the Basic Reading Level that combines level 0/1 looks and sounds really good to me.

This version Voices Screen Text & Object Changes, and fully support mouse, touch, and keyboard access to readable objects and interactive objects.

The dev-version link is:
https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-dev.7/phet/gravity-force-lab-basics_en_phet.html?selfVoicing=levels

**Comment by terracoda** [comment]
The above dev link is for what we are now calling Paradigm 2.
I will start a new git hub issue with a slight change (fewer readable blocks added to navigation order). We call this next version Paradigm 3.

**Comment by jessegreenberg** [comment]
THis paradigm is done, and we are moving on to #205. Im going to close this issue.
